.gitignore

```diff
@@ -1,4 +1,4 @@
 target
 *.log
 ignore*
-.DS_Store
+.DS_Store
```

Comment:
Unnecessary change, let's keep the convention of having a blank new line after each file. VSCode can configured to do that through `insertFinalNewline`.